ui/src/VoxettaApp.js

```diff
@@ -62,11 +103,35 @@ export class VoxettaApp extends LitElement {
         switch (this.state) {
             case States.RECORD_PAGE:
                 return html`
-                    <vox-user-icon 
-                        @enter-form="${() => { this.state = States.USER_FORM }}">
-                    </vox-user-icon>
-                    <vox-prompts></vox-prompts>
-                    <vox-record-button></vox-record-button>
+                    <div class="header">
+                        <vox-user-icon 
+                            @enter-form="${() => { this.state = States.USER_FORM }}">
+                        </vox-user-icon>
+                    </div>
+                    <div class="prompts">
+                        <vox-prompts
+                            .promptState=${this.prompt}></vox-prompts>
+                        <vox-sound-wave 
+                            .isRecording=${this.isRecording} 
+                            .audioStream=${this.audioStream}>
+                        </vox-sound-wave>
+                    </div>
+                    <div class="buttons">
+                        <div class="button-container"></div>
+                        <div class="record-button-container">
+                            <vox-record-button
+                                @record-state-change="${(e) => { 
+                                    this.isRecording = e.detail.isRecording;
+                                    this.audioStream = e.detail.audioStream; }}"
+                                @next-prompt="${() => { this.promptState = !this.promptState }}">
```

Comment:
Make these properties of the class. Else when the component rerenders a new function is created and the vox-record-button will rerender even though the functions function has not changed.

Also handlers generally follow the format @verb-object
@change-prompt
@skip-prompt
@start-recording

Reply:
Sorry I'm a little confused--can you please clarify which you are referring to make properties of the class? also switched to verb-object style

Reply:
Ohh sorry, the eventHandler functions. 
@change-prompt=${this.handleChangePrompt}

instead of:
@change-prompt=${(e) => doSomething(e)}

Reply:
Changed for @change-prompt